### PR TITLE
Add various missing post install hooks

### PR DESCRIPTION
**Pull request by Dennisbonke**
This PR adds several missing post install hooks to glib and related packages. Fixing this allows for a better experience in Managarm.

### CODE REVIEW

**Comment by avdgrinten** [comment]
Do all these operations really take a few seconds? If not, I'd just delete the sentence (we don't need it at every hook, we just have it in the fc-cache invocation since that can truly take a long time.

File: bootstrap.d/dev-libs.yml
Comment (avdgrinten) [inline bootstrap.d/dev-libs.yml:248]: Why is this a shell invocation and not just a argument array (same for the other instances)? AFAICS the previous patch didn't work because it was missing a comma in the arg array. But `['glib-compile-schemas', '/usr/share/glib-2.0/schemas']` should work, right?

Comment (Dennisbonke) [reply bootstrap.d/dev-libs.yml:248]: You might be on to something there. I’ll change it around and test it. If that works, I’ll use that over shell invocations.

**Comment by Dennisbonke** [comment]
This still doesn't work properly unfortunately. Marking as draft pending further investigation.

**Comment by Dennisbonke** [comment]
Some of these are shell invocations, after a long battle that is how I got it to work. At this point in time all hooks _should_ work, so this can be merged now.

**Comment by Dennisbonke** [comment]
At the request of @ArsenArsen, I've reworded several commits so they reflect the changes more accurately. It _should_ be good to go now, but if @avdgrinten and @ArsenArsen want to take another look before hitting that merge button that would be great.

**Review by avdgrinten — APPROVED** [review]

**Pull request merged**
